ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -285,7 +285,6 @@ private void createBanner() {
         if (mVungleManager == null || !pendingRequestBanner.get())
             return;
 
-        mVungleManager.cleanUpBanner(mPlacementForPlay);

```

Comment:
I removed this line because in VungleManger#getVungleNativeAd() there's a duplicate cleanUpBanner() call.